lib/matplotlib/tests/test_artist.py

```diff
@@ -176,6 +177,26 @@ def test_remove():
     assert_true(ax.stale)
 
 
+@image_comparison(baseline_images=["default_edges"], remove_text=True,
+                  extensions=['png'], style='default')
+def test_default_edges():
+    with mpl.rc_context({'patch.linewidth': None}):
+        fig, [[ax1, ax2], [ax3, ax4]] = plt.subplots(2, 2)
+
+        ax1.plot(np.arange(10), np.arange(10), 'x',
+                 np.arange(10) + 1, np.arange(10), 'o')
+        ax2.bar(np.arange(10), np.arange(10))
+        ax3.text(0, 0, "BOX", bbox=dict(boxstyle='sawtooth'))
```

Comment:
Really? It looks dotted to me. Also, I thought `remove_text=True` removed all of the text for the test? I didn't see the black "BOX" text in the blue box at first.